app/api/sets/[id]/route.ts

```diff
@@ -0,0 +1,90 @@
+import { NextResponse } from 'next/server';
+import { SetRepository } from '../../repositories/Sets';
+import { SetService } from '../../services/SetsService';
+
+const setService = new SetService(new SetRepository());
+
+export async function GET(
+    request: Request,
+    { id }: { id: string }
+) {
+    try {
+        const set = await setService.getSet(id);
+
+        if (!set) {
+            return NextResponse.json({ error: "Set not found" }, { status: 404 });
```

Comment:
mantengamos una convencion para los 404, el primer parametro que sea vacio
```suggestion
            return NextResponse.json({}, { status: 404 });
```

Reply:
changes applied